### PR TITLE
Feat/cart management

### PR DESCRIPTION
**Pull request by sonishreyas**
## Change logs

Cart management page added. Also added a grid layout.
Since many of my components in the component library are not merged I have kept a copy o them in this repo. Once they are merged I will remove them from this repo.

## Screenshot
image
 

## Preview Link
https://deploy-preview-2--ripple-store.netlify.app/pages/cart/cart.html

### CODE REVIEW

File: pages/cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Shreyas c. Soni">
+    <!-- <link rel="stylesheet" href="https://ripple-ui.netlify.app/css/components.css" /> -->
+    <link rel="stylesheet" href="../../css/components.css">
```
Comment (Rockyraven) [inline pages/cart/cart.html:10]: remove that unwanted code

**Comment by Rockyraven** [comment]
all goods for me

File: pages/cart/cart.html (hunk 1 above)
Comment (sonishreyas) [reply pages/cart/cart.html:10]: Yes will remove it once all the components of my component library are merged so I can refer that CSS directly

**Review by sach10-create — APPROVED** [review]
Good to merge

**Pull request merged**